src/SearchBar/index.js

```diff
@@ -90,6 +90,7 @@ const SearchIcon = styled.div`
   font-size: ${(props) => getIconFontSize(props)};
   background-color: whitesmoke;
   transition: all 250ms ease;
+  opacity: ${(props) => (props.disabled ? "0.6" : "1")};
```

Comment:
Remove all opacity properties from SearchBar component. Just the Chip component changes opacity on disabled true/false.